Product/DBScripts/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.connectopensource</groupId>
         <artifactId>Product</artifactId>
-        <version>5.0-SNAPSHOT</version>
+        <version>5.0.0-RC1</version>
```

Comment:
All of the changes from snapshot back to RC1 need to be reverted.  Looks like you didn't pull the latest before submitting the PR.